spec/requests/api/blueprints_spec.rb

```diff
@@ -351,7 +355,10 @@
         :action   => "edit",
         :resource => {
           :bundle => {
-            :service_dialog => nil
+            :service_catalog => {
```

Comment:
Same here - it was my understanding that even though the UI might send the whole bundle every time, the API shouldn't need it to be specified. Unless I am missing osmething?